### ISSUE TITLE
DL3026 is not detecting violations on `COPY --from` instructions

### ISSUE THREAD

**Issue by phansys** [hadolint/hadolint#1054]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Violation with this setup:

```dockerfile
# Dockerfile

# ...

COPY --from=scratch /opt /opt
```
```shell
hadolint --trusted-registry my-company.com:500 Dockerfile
```

### Actual behavior

No violations emitted.

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```shell
Haskell Dockerfile Linter 2.12.0
```

**Comment by m-ildefons** [comment]
This issue was duplicated here: https://github.com/hadolint/hadolint/issues/1108
Since the discussion in the other issue was more productive and highlighted a few other cases where the (un-)trusted registries needed to be checked, I'll close this issue in favor of the other one - even though this issue is older.
Nevertheless, `COPY --from=...` is going to be checked against (un-)trusted registries in the next Hadolint release.
